src/brain-gain/main.css

```diff
@@ -31,10 +39,48 @@ core-navbar a:hover,
 core-navbar a:active {
   transform: scale(1.4);
 }
+core-navbar b:hover,
+core-navbar b:active {
+  transform: scale(1.2);
+}
 core-navbar > div {
   padding: 8px;
 }
 
+/* best scores for all games */
+.best-scores {
+  position:relative;
+  justify-content: center;
+  margin-right: 3px;
+  padding: 6px;
+  border-radius: 3px;
+  background-color: #f4abba;
+  color: #f5f5f5;
+}
+
+.best-game-list {
+  display: flex;
+  justify-content: space-around;
+  position: relative;
+  width: 500px;
+  height: 200px;
+}
+
+.best-game-scores {
+  display: inline-block;
+  justify-content: center;
+  position: relative;
+  text-align: center;
+  margin-top: 20px;
+  font-size: 30px;
+  color: #595959;
+}
+
+.best-game-img {
```

Comment:
you can use `aspect-ratio: auto;` in css to stop it from stretching like this. :)
![image](https://user-images.githubusercontent.com/79813230/192944603-72970843-1e53-49d1-ae88-6aede88d8229.png)

Reply:
resolved